tfutils/base.py

```diff
@@ -68,8 +86,20 @@
                                   'save_to_gfs': (),
                                   'do_save': True})
 
-
-DEFAULT_LOAD_PARAMS = frozendict({'do_restore': True, 'load_param_dict': None, 'load_step': True, 'from_ckpt': None})
+DEFAULT_PARAMS = frozendict({
+    'dont_run': False,
+    'skip_check': False,
+    'model_params': {},
+    'train_params': {},
+    'validation_params': {},
+    'log_device_placement': False,
+    'inter_op_parallelism_threads': 40,
+    'save_params': dict(DEFAULT_SAVE_PARAMS),
```

Comment:
@alexandonian should we be using frozendict here? and in similar places in the defaults?